pkg/scale/errors.go

```diff
@@ -0,0 +1,20 @@
+package scale
+
+import "errors"
+
+var (
+	ErrUnsupportedDST                  = errors.New("must be a pointer to a destination, unsupported dst")
```

Comment:
What does `DST` mean? Is it destination? Should we use `Destination` instead, if this is so?